snowflake-cortex/app/manifest.yml

```diff
@@ -0,0 +1,12 @@
+# For more information on creating manifest, go to https://docs.snowflake.com/en/developer-guide/native-apps/creating-manifest
+manifest_version: 1
+
+artifacts:
+  setup_script: setup_script.sql
+  readme: README.md
+  default_streamlit: core.dashbaord
```

Comment:
This either needs to be `core.ui` or we need to change the setup script to create `core.dashboard`

Reply:
addressed in the latest commit